server/app/controllers/HomeController.java

```diff
@@ -51,11 +47,13 @@ public HomeController(
   public CompletionStage<Result> index(Http.Request request) {
     Optional<CiviFormProfile> maybeProfile = profileUtils.currentUserProfile(request);
 
```

Comment:
The `loginForm` endpoint below is gone. We replace it with the endpoint that the login form directs users to when they used to click "Continue as Guest".

Reply:
Edit: I wrote this comment before I introduced the feature flag. The `loginForm` endpoint **will** be done once this change is finalized and the flag is removed.